image/scanner/konflux/Dockerfile

```diff
@@ -0,0 +1,62 @@
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8-minimal
+ARG BASE_TAG=latest
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
+
+ENV CGO_ENABLED=1
+ENV GOFLAGS=""
+ENV CI=1
+
+COPY . /src
+WORKDIR /src
+
+RUN unzip -j blob-repo2cpe.zip -d image/scanner/dump/repo2cpe && \
+    unzip -j blob-k8s-definitions.zip -d image/scanner/dump/k8s_definitions && \
+    unzip -j blob-nvd-definitions.zip -d image/scanner/dump/nvd_definitions
+
+RUN echo "version: $(git describe --tags --abbrev=10 --dirty --long)"; git status; go build -trimpath -ldflags="-X github.com/stackrox/scanner/pkg/version.Version=$(git describe --tags --abbrev=10 --dirty --long)" -o image/scanner/bin/scanner ./cmd/clair
+
+# Replace genesis manifests file in the source code with the one generated at
+# the point when the dump was taken.  This is to avoid discrepancy between other
+# files of the dump and the manifest.
+COPY ./blob-genesis_manifests.json image/scanner/dump/genesis_manifests.json
+
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
```

Comment:
[not about this line]
There seems to be `SHELL ["/bin/sh", "-o", "pipefail", "-c"]` defined in both upstream and downstream dockerfiles.
I don't know if there's a reason to define `pipefail` at the container level: I don't see any use of pipes below and I see that `entrypoint.sh` with its dependencies does call `set -...o pipefail` explicitly.

If you omit `SHELL ["/bin/sh", "-o", "pipefail", "-c"]` here, could you also please kill it from upstream dockerfiles for consistency? Later on, someone may take care of the downstream ones.

Alternatively let's add `SHELL ...` here (again: for consistency) and have someone eventually deal with it later.

Reply:
I'd rather avoid touching the upstream dockerfiles if I can help it, so I'll add the `SHELL` line to the konflux dockerfile.

Reply:
d77527a